listenbrainz/webserver/static/js/src/follow/FollowerFollowingModal.tsx

```diff
@@ -112,23 +71,14 @@ export default class FollowerFollowingModal extends React.Component<
           {activeModeList.map((listEntry: ListenBrainzUser) => {
             return (
               <>
```

Comment:
My pet peeve :p
Can we remove this empty tag?

Reply:
oops